lib/assessment.md

```diff
@@ -83,7 +101,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+color: yellow;
+font-family: sans-serif;
```

Comment:
Sorry - cursive. It's given in the inline style, and that trumps anything written in the stylesheet.